README.md

```diff
@@ -133,6 +122,14 @@ Here's a rough roadmap for the project, with items near the top of the list more
   * [ ] Provide a web interface analogous to the command line interface. Much like GitHub/Gitea/Forgejo make git more approachable, you shouldn't have to pay a command line knowledge tax in order to create, share, and query an `ayb` database.
   * [ ] Explore people's public datasets. Beyond simplifying the command line, platforms like GitHub also make it easier to find a user's publicly shared repositories, follow along in their work, and fork a copy for your own exploration. That same experience should be possible for `ayb`-hosted databases.
 
+## FAQ
+* When should I use ayb? when would I be better served by datasette? What about datasette?
```

Comment:
I really like providing links to related projects, especially when writing open source software.  If I landed on this page, and ayb wasn't quite the thing I was looking for, but datasette was, I'd really appreciate you.